sdk/cosmos/azure-cosmos-spark_3_2-12/Samples/DatabricksLiveContainerMigration/README.md

```diff
@@ -0,0 +1,76 @@
+# Live Migrate Azure Cosmos DB SQL API Containers data with Spark Connector and Azure Databricks
+
+This sample shows how to live migrate between Azure Cosmos DB SQL API containers using the [Azure Cosmos DB Spark Connector](https://docs.microsoft.com/azure/cosmos-db/sql/create-sql-api-spark). 
+
+# Setup Azure Databricks
+
+## Prerequisites
+
+* [Provision an Azure Databricks cluster](https://docs.microsoft.com/azure/databricks/scenarios/quickstart-create-databricks-workspace-portal?tabs=azure-portal).
+
+* Ensure it has network access to your source and target Azure Cosmos DB Containers.
+
+
+## Provision a Spark cluster
+
+Select the latest Azure Databricks runtime version which supports Spark 3.0 or higher.
```

Comment:
do we have recommendation for worker type and the # of workers?

Reply:
Not that I am aware of, but the default config (2 min workers and 8 max workers with autoscale of workers enabled) is usually sufficient for most tasks.